lib/OpenQA/WebAPI.pm

```diff
@@ -298,7 +306,7 @@ sub startup ($self) {
 
     # api/v1/job_groups
     $api_public_r->get('/job_groups')->name('apiv1_list_job_groups')->to('job_group#list');
-    $api_public_r->get('/job_groups/<group_id:num>')->name('apiv1_get_job_group')->to('job_group#list');
```

Comment:
I guess the rout name (`apiv1_get_job_group`) could stay.

Reply:
This is just a proof of concept with two routes. We don't need detailed routes likr this anymore when we move all API routes to openAPI

Reply:
Ok, it is just that we use these names with the `url_for` helper all over the place. I'm wondering how this would look like then.

Reply:
It could be that operationId is what's used for the route name. I would have to check.